packages/smart-contracts/src/contracts/lib/SafeERC20.sol

```diff
@@ -49,13 +49,6 @@ library SafeERC20 {
     return result;
   }
   
-  /**
-    * @dev Deprecated. This function has issues similar to the ones found in
```

Comment:
Why are we removing this? It's not the case anymore?

Reply:
I think the deprecation does not concern us because we approve the max every time... But I can put it back.

Reply:
It's fine, just trying to understand the context.